pkg/cmdutil/util.go

```diff
@@ -0,0 +1,31 @@
+package cmdutil
+
+import "strings"
+
+func GetNestedStringField(inputMap map[string]interface{}, selector string) (string, bool) {
```

Comment:
This might be ever-so-slightly cleaner if we accepted `keys ...string` instead of having to parse a `selector` format.

Reply:
I'll resolve the other comments on this function since I went ahead with using `gjson`. I figured it makes things a lot cleaner and is less code for us to maintain and test. Thanks for recommending! :)  05e1710